lib/RocketMap.php

```diff
@@ -48,12 +56,27 @@ public function get_active($eids, $swLat, $swLng, $neLat, $neLng, $tstamp = 0, $
             $pkmn_in = substr($pkmn_in, 0, -1);
             $conds[] = "pokemon_id NOT IN ( $pkmn_in )";
         }
-
+        $float = $db->info()['driver'] == 'pgsql' ? "::float" : "";
+        if (!empty($miniv) && !is_nan((float)$miniv) && $miniv != 0) {
+            if (empty($exminiv)) {
+                $conds[] = '((individual_attack + individual_defense + individual_stamina)' . $float . ' / 45) * 100 >= ' . $miniv;
+            } else {
+                $conds[] = '(((individual_attack + individual_defense + individual_stamina)' . $float . ' / 45) * 100 >= ' . $miniv . ' OR pokemon_id IN(' . $exminiv . ') )';
+            }
+        }
+        if (!empty($minlevel) && !is_nan((float)$minlevel) && $minlevel != 0) {
+            if (empty($exminiv)) {
+                $conds[] = '(cp_multiplier >= ' . $this->cp_multiplier[$minlevel];
+            } else {
+                $conds[] = '(cp_multiplier >= ' . $this->cp_multiplier[$minlevel] . ' OR pokemon_id IN(' . $exminiv . ') )';
+            }
+        }
         return $this->query_active($select, $conds, $params);
     }
 
-    public function get_active_by_id($ids, $swLat, $swLng, $neLat, $neLng)
+    public function get_active_by_id($miniv, $minlevel, $exminiv, $ids, $swLat, $swLng, $neLat, $neLng)
```

Comment:
This seems to be in the wrong order, `$ids` should be the first parameter

Reply:
Good catch updated this